src/openzeppelin/tests/test_account.cairo

```diff
@@ -52,7 +52,7 @@ fn SIGNED_TX_DATA() -> SignedTransactionData {
     }
 }
 
-fn setup_dispatcher(data: Option<@SignedTransactionData>) -> IAccountDispatcher {
+fn setup_dispatcher(data: Option<@SignedTransactionData>) -> AccountABIDispatcher {
```

Comment:
we can now use the `deploy` util that was merged with [reentrancy guard](https://github.com/OpenZeppelin/cairo-contracts/pull/590/files#diff-2097b4c219a2e217e3df21a7c195b41a8590d29bedcd81982e795f7983a9a645R1)